### PR TITLE
Term entry sdl bigbang

### LINKED ISSUE #1804: [Term Entry] General SDLC: Big Bang
### Reason/inspiration (optional)

The [Software Development Lifecycle](https://www.codecademy.com/resources/docs/general/software-development-life-cycle) Model to cover:

- Big Bang: Named after the physical theory, this model involves creating a product through available resources for technical development. There is little planning or documenting.

Please write a new entry and expand on the above introductory paragraph under:  content/general/concepts/software-development-life-cycle/terms/ about the Big Bang model.

To get started with the PR and entry file:

Under content/general/concepts/software-development-life-cycle/terms/, create a new folder called `big-bang`. Then create a new file called `big-bang.md` under content/general/concepts/software-development-life-cycle/terms/`big-bang`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for General Concepts - SDLC Math Functions: Big Bang Model
- Closes #1804 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **Big Bang** model is a [Software Development Life Cycle](https://www.codecademy.com/resources/docs/general/software-development-life-cycle) (SDLC) model without specific processes.
+
+The Big Bang model requires no or minimal planning. Instead, the development starts with acquiring the inputs, such as:
+
+- time,
```
Comment (SSwiniarski) [inline content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:19]:
```suggestion
- Time.
```

File: content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- effort and
```
Comment (SSwiniarski) [inline content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:20]:
```suggestion
- Effort.
```

File: content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- resources.
```
Comment (SSwiniarski) [inline content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:21]:
```suggestion
- Resources.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla Strong entry, I only had a few comments. I have a question about the image. Is it your own work, or did you source it from somewhere?

**Comment by cslylla** [comment]
@SSwiniarski Thank you, I accepted the changes. I made the image with Figma. I used icons from [Material Design](https://fonts.google.com/icons) and [Icons8](https://icons8.com/). I tried to use the same colours that Codecademy uses.

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good for a second review. Complements on the image.

File: content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Big Bang'
```
Comment (caupolicandiaz) [inline content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:2]:
```suggestion
Title: 'Big Bang Model'
```

Comment (caupolicandiaz) [reply content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md:2]: We have decided to updated all of the SLDC terms to include model in the title and the file name per https://github.com/Codecademy/docs/pull/2177. Please update this entry to match.

@yangc95

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla Looks good. Just a couple edits to match our new standards for this topic. Please tag me when you've finished making the changes. Thanks

**Comment by cslylla** [comment]
@caupolicandiaz I updated the file and folder name to include "model" in it. I also made a minor change to the document: I changed  "model" to bold in the first sentence.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good

**Comment by SSwiniarski** [comment]
@cslylla Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/general/software-development-life-cycle/big-bang-model

image

**Pull request merged**